packages/gestalt/src/AvatarGroup.js

```diff
@@ -16,9 +16,9 @@ type Size = 'xs' | 'sm' | 'md' | 'fit';
 type UnionRefs = HTMLDivElement | HTMLAnchorElement;
 type Props = {|
   /**
-   * Label for the component used for screen readers.
+   * Label for screen readers to announce Icon.
```

Comment:
Does AvatarGroup show icons?

Reply:
I liked better the other description. Missed changing icon with avatargroup

Reply:
DONE